website/source/intro/getting-started/build-image.html.md

```diff
@@ -314,105 +317,203 @@ amazon-ebs output will be in this color.
 ==> amazon-ebs: Waiting for AMI to become ready...
 ```
 
-### A windows example
-
-Note that this uses a larger instance.  You will be charged for it. Also keep 
-in mind that using windows AMIs incurs a fee that you don't get when you use 
-linux AMIs.
-
-You'll need to have a boostrapping file to enable ssh or winrm; here's a basic 
-example of that file.
-
-```
-# set administrator password
+### A Windows Example
+
+As with the GNU/Linux example above, should you decide to follow along and
+build an AMI from the example template, provided you qualify for free tier
+usage, you should not be charged for actually building the AMI.
+However, please note that you will be charged for storage of the snapshot
```

Comment:
> ... this is awesome and I love the polish you've added here. I tested the example and it works like a dream.

Great! Thanks. So I was a little worried about some of the changes - these things can be quite subjective - but I'm really glad your happy with this.

> I'm pretty sure you get charged for using windows images, period, in a way that isn't the case for linux.

So I think the original value for the name field you had used in the `source_ami_filter` was returning an image that required a larger instance and so wasn't eligible for the free tier. Also, a lot of the Windows AMI's have SQL server or some other software bundled that you do pay for... 

>  Also, windows doesn't work on the small free-tier t2.micro images...

You can definitely run Windows with the free-tier t1.micro and t2.micro instances. The [templates I used to test with](https://github.com/DanHam/packer-testing/tree/master/aws/windows) specify a t2.micro type. I'm so cheap! 😃 ...

If you navigate to the [AWS AMI management page](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;ownerAlias=amazon;name=Windows_Server-2012-R2_RTM-English-64Bit-Base-2017.09.13;sort=name) you will see the image returned by the new `source_ami_filter` we are now using in the example template - note the link has the filters for finding the AMI pre-configured so you should only see the `Windows_Server-2012-R2_RTM-English-64Bit-Base-2017.09.13` AMI we are using as our source.

If you select the image and click on the Launch button, the next page you see will be the `Step 2: Choose an Instance Type` page. You should see the t2.micro image type listed along with the green 'Free Tier Eligible' badge... so definitely free!

Note that the `source_ami_filter` we are now using should work for any region. I usually swap the region to `eu-west-2` rather than `us-east-1` and I get the same image - Amazon ensure their AMI names are consistent across regions.

Reply:
Oh hey so you did!  That's wonderful.  You'd better watch out... if you keep improving my windows and ecosystem knowledge and I'm gonna have to start tagging you in windows PRs